src/auto_scheduler/feature.cc

```diff
@@ -669,7 +669,7 @@ class PerStoreFeatureExtractor : public StmtExprVisitor {
     math_op_counter(node->value);
     std::vector<float> mem_bytes_list;
     std::vector<float> compute_ops_list;
-    int cur_compute_ops;
+    double cur_compute_ops;
```

Comment:
Why moving this to double?

Reply:
int32 will overflow.

Reply:
What about int64? I don't mind if not, but when I read `double` I thought that it could be a non-integer value (e.g., 0.1234)